pkgs/tools/misc/zalgo/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zalgo-unstable";
+  version = "2020-08-26";
+
+  src = fetchFromGitHub {
+    owner = "lunasorcery";
+    repo = "zalgo";
+    rev = version;
```

Comment:
How it works for you? There is no tags in the repo.

Reply:
Sorry, I was just running `nix-build -A zalgo` without updating the source!

I misunderstood one of the earlier review comments. I think dedf0de addresses this.

Reply:
I looked at some other examples and updated again, which I think is more in line with the established convention and notes in the nixpkgs manual.